cpp/cmake_modules/ThirdpartyToolchain.cmake

```
@@ -1019,12 +1019,13 @@ if(ARROW_USE_BOOST)
     # Find static boost headers and libs
     set(Boost_USE_STATIC_LIBS ON)
   endif()
+  if(ARROW_BOOST_REQUIRE_LIBRARY)
+    set(ARROW_BOOST_COMPONENTS "COMPONENTS;system;filesystem")
```

Could you use normal list literal form instead of string representation form for easy to read? 

```suggestion
    set(ARROW_BOOST_COMPONENTS COMPONENTS system filesystem)
```